src/scripts.js

```diff
@@ -166,32 +164,27 @@ function compareDayActivity() {
   `
 }
 
-function hydrationGraph(date, ounces) {
-
+function hydrationGraph(hydrationData) {
+console.log(hydrationData)
   var chart = new CanvasJS.Chart("chartContainer", {
     title:{
       text: "Your Weekly Hydration Data in Ounces"
     },
     data: [
-    {
+      {
       // Change type to "doughnut", "line", "splineArea", etc.
-      type: "column",
-      dataPoints: [
-        { label: date[0],  y: ounces[0] },
-        { label: date[1],  y: ounces[1] },
-        { label: date[2],  y: ounces[2] },
-        { label: date[3],  y: ounces[3] },
-        { label: date[4],  y: ounces[4] },
-        { label: date[5],  y: ounces[5] },
-        { label: date[6],  y: ounces[6] }
-      ]
-
-    // date.forEach((day, index) => {
-    //   data.dataPoints.push({ label: date[index],  y: ounces[index] },)
-    // })
-  }
+        type: "column",
+        dataPoints: [
+          { label: hydrationData[0].date,  y: hydrationData[0].ounces },
+          { label: hydrationData[1].date,  y: hydrationData[1].ounces },
+          { label: hydrationData[2].date,  y: hydrationData[2].ounces },
+          { label: hydrationData[3].date,  y: hydrationData[3].ounces },
+          { label: hydrationData[4].date,  y: hydrationData[4].ounces },
+          { label: hydrationData[5].date,  y: hydrationData[5].ounces },
+          { label: hydrationData[6].date,  y: hydrationData[6].ounces }
+        ]
+      }
```

Comment:
This now just displays the correct information based on the object that was passed in, instead of two seperate parameters.